web/components/issues/issue-modal/form.tsx

```diff
@@ -179,6 +182,28 @@ export const IssueFormRoot: FC<IssueFormProps> = observer((props) => {
 
   const projectDetails = getProjectById(projectId);
 
+  // executing this useEffect when the parent_id coming from the component prop
+  useEffect(() => {
+    const parentId = watch("parent_id") || undefined;
+    if (!parentId) return;
+    if (parentId === selectedParentIssue?.id || selectedParentIssue) return;
+
+    const issue = getIssueById(parentId);
+    if (!issue) return;
+
+    const projectDetails = getProjectById(issue.project_id);
+    if (!projectDetails) return;
+
+    setSelectedParentIssue({
```

Comment:
TODO:// This should be done when user clicks update issue, not when the parent id is selected

Reply:
This seems to be fine, i mistook it for a store action, it is just a local state